client/blocks/comment-detail/style.scss

```diff
@@ -12,9 +12,16 @@
 		}
 	}
 
-	&.is-collapsed:hover {
-		box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20% );
-		z-index: 1;
+	&.is-collapsed {
+		&:hover {
+			box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20% );
+			z-index: 1;
+		}
+
+		.accessible-focus &:focus {
+			box-shadow: 0 0 0 1px $blue-medium, 0 0 0 3px $blue-light;
+			z-index: 1;
```

Comment:
I'd like the value to be in _z-index.scss, so folks don't begin to use bare values elsewhere. It won't be quite right since this may be used in many places and might have different parent contexts, but I think placing it in the root context should be fine for now.